### PR TITLE
{vis}[GCCcore/9.3.0,intel/2018a] GLFW v3.2.1, GLFW v3.3.2

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GLFW/GLFW-3.2.1-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+sources = ['%(version)s.tar.gz']
+checksums = ['e10f0de1384d75e6fc210c53e91843f6110d6c4f3afbfb588130713c2f9d8fe8']
+
+builddependencies = [
+    ('CMake', '3.12.1'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('X11', '20180131'),
```
Comment (Micket) [inline easybuild/easyconfigs/g/GLFW/GLFW-3.2.1-intel-2018a.eb:22]: Is there really no dependency on Mesa/libGLU here?

Comment (SimonPinches) [reply easybuild/easyconfigs/g/GLFW/GLFW-3.2.1-intel-2018a.eb:22]: I think not (but am not an expert).  You can see more here: https://www.glfw.org/docs/3.3/compile.html

If you think it safer to add Mesa as an explicit dependency anyway, then I'm happy to do so.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node356.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/12e903cfa0beb9ccdeb7d143fdcdbba4 for a full test report.

File: easybuild/easyconfigs/g/GLFW/GLFW-3.2.1-intel-2018a.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/g/GLFW/GLFW-3.2.1-intel-2018a.eb:22]: I'm not an expert either, but checking the link above and the sources it seems Mesa can be used at runtime if you build with `-DGLFW_USE_OSMESA`, as Mesa is built with gallium-osmesa.
however, in that case it is built without X11 support, so I think it is fine as it is without Mesa.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**